### PR TITLE
RDS/JDJ/Rule1-4

### PR DESCRIPTION
**Pull request by JacksonJ-KC**
Rule ID is my suggested new numbering that has not yet been incorporated into MasterListofRules. See [Performance Calculations TCD Spreadsheet](https://onedrive.live.com/edit.aspx?resid=796769604E905084!1713&cid=796769604e905084&authkey=!ACtFTkgcGDgG1Qk&CT=1701831975786&OR=ItemsView) for my suggested changes to rules and ID.

### CODE REVIEW

File: docs/section1/Rule1-4.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+**Applicability Checks:** None
+
+**Function Calls:** None
+
+
+## Rule Logic:
+- Get the Performance Cost Index-Target from the output of the proposed model: `output_pci_target = P_RMD.output.performance_cost_index_target`
+- Get the Performance Cost Index from the output of the proposed model: `output_pci = P_RMD.output.performance_cost_index`
+**Rule Assertion**
```
Comment (claperle) [inline docs/section1/Rule1-4.md:22]: Same question as for 1-3 about which RMD to collect this information from.

File: docs/section1/Rule1-4.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+**Rule Description:** The PCI shall be less than or equal to the PCIt when calculated in accordance with Standard 90.1 2019, Section 4.2.1.1  
+**Rule Assertion:** Options are Pass/Fail  
+**Appendix G Section:** None  
+**90.1 Section Reference:** 4.2.1.1  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** RMD
+
+**Applicability Checks:** None
```
Comment (claperle) [inline docs/section1/Rule1-4.md:14]: This does not account for renewables but that is not clear from the rule description or applicability.

Comment (JacksonJ-KC) [reply docs/section1/Rule1-4.md:14]: Do you mean it does not account for the limit on renewables? PCI does include renewables. My understanding is this rule is making sure the project complies without consideration for the limit on renewables, then if the renewables exceed the 5% limit Rule 1-5 becomes applicable.

Comment (claperle) [reply docs/section1/Rule1-4.md:14]: Okay, I see. That makes sense.

**Review by KarenWGard — APPROVED** [review]

**Review by claperle — APPROVED** [review]

**Pull request merged**